qubesbuilder/plugins/source/salt/yaml-dumper

```diff
@@ -0,0 +1,133 @@
+#!/bin/python3
+#
+# The Qubes OS Project, http://www.qubes-os.org
+#
+# Copyright (C) 2015 Jason Mehring <nrgaway@gmail.com>
+# Copyright (C) 2022 Frédéric Pierret (fepitre) <frederic@invisiblethingslab.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+#
+# Dump a YAML configuration file to key = value pairs
+#
+
+import argparse
+import collections
+import os
+import sys
+import yaml
+
+from yaml.parser import ParserError
+from yaml.reader import ReaderError
+
+__version__ = "1.0.0"
+
+# prefer C bindings over python when available
+BaseLoader = getattr(yaml, "CLoader", yaml.Loader)
+
+
+def _ordered_load(stream, Loader=BaseLoader, object_pairs_hook=collections.OrderedDict):
+    class OrderedLoader(Loader):
+        pass
+
+    def construct_mapping(loader, node):
+        loader.flatten_mapping(node)
+        return object_pairs_hook(loader.construct_pairs(node))
+
+    OrderedLoader.add_constructor(
+        yaml.resolver.BaseResolver.DEFAULT_MAPPING_TAG, construct_mapping
+    )
+
+    return yaml.load(stream, OrderedLoader)
```

Comment:
```suggestion
    return yaml.safe_load(stream, OrderedLoader)
```

Reply:
It looks like `safe_load` does not support providing Loader.